graphql/database/missions/utils.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-nested-ternary */
+/* eslint-disable no-param-reassign */
+import UserModel from '../users/UserModel'
+import {
+  getPermissionsOverride,
+  MISSIONS_OVERRIDE,
+} from '../../utils/permissions-overrides'
+
+const override = getPermissionsOverride(MISSIONS_OVERRIDE)
+export const buildSquadMemberDataFromMissionDoc = missionDoc => {
```

Comment:
Don't export if unneeded. (For unit tests, try to test against the default export to ignore implementation details.)